dir/path_darwin.go

```diff
@@ -0,0 +1,15 @@
+//go:build darwin
+// +build darwin
```

Comment:
It is not necessary to specify go build flags. Files names with `_darwin.go` suffix are special ones similar to `_windows.go`, `_test.go` and etc.

Reply:
Updated.  Learned a lot.